domain/domain.api.php

```diff
@@ -36,10 +36,10 @@ function hook_domain_load(array $domains) {
  * To issue a redirect, as in the case of Domain Alias, set a redirect
  * property to a valid response code (301 or 302).
  *
- * @param DomainInterface $domain
+ * @param \Drupal\domain\DomainInterface $domain
  *   A domain object defined by Drupal\domain\DomainInterface.
  */
-function hook_domain_request_alter(DomainInterface &$domain) {
+function hook_domain_request_alter(\Drupal\domain\DomainInterface &$domain) {
```

Comment:
I am in two worlds about it. Core is inconsistent. e.g. see `hook_block_view_alter()`